### DISCUSSION TITLE
Add support for using custom domain for Altinn Studio Apps

### DISCUSSION THREAD

**Discussion by pckofstad** [Altinn/altinn-platform#728] [Ideas]
### Description

Udir would like to have a **branded experience** for the students/parents using the PBU platform. This includes a clean custom udir.no subdomain for the app (example: undersokelser.udir.no). As we understand, this is not supported in Altinn Studio today. We would like to put this out there as a future feature request. We think there will be more teams that would consider this a “nice to have” feature, that will raise the bar and make the apps look even more professional. 

We know that as of Q1 2023 there is a lot of higher priority items in the backlog that might make more sense, therefore this can be considered a “low priority” feature request from our side. 

I received a comment from Altinn servicedesk that summarize the some of the technical changes that needs to be done. 

> Custom domener er ikke noe vi støtter i dag.
> Vi har hatt en liten runde internt for å avdekke hvilke områder som blir problematiske, noen kan ha en enkel fiks mens andre krever store endringer.
> * Cookies utstedes til *.altinn.no disse vil da ikke bli med hvis bruker har *.udir.no i sin nettleser
> * Knytninger i app-frontend app-lib til domenenavnet f.eks [her](https://github.com/Altinn/app-frontend-react/blob/main/src/utils/urls/urlHelper.ts)
> * Custom helm chart med nye FQDN referanser for udir apps
> * TLS private key for .[udir.no](http://udir.no/) må legges inn i apps clusteret til udir som altinn drifter

**Comment by Herskis** [comment]
@altinnadmin Her tenker jeg du må være med på diskusjonen!

**Comment by altinnadmin** [comment]
@Herskis We have bigger ambitions than just support for custom domains... 😁

It sounds like this should be one of many epics that will be needed to realize this roadmap element: https://github.com/orgs/digdir/projects/8/views/2?layout=roadmap&pane=issue&itemId=24023094
